### PR TITLE
chore(server): update license for 1.2.0

### PR DESCRIPTION
**Pull request by SunnyBoy-WYH**
## Purpose of the PR
 https://github.com/hugegraph/actions/issues/6

## Verifying these changes

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [ ]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [ ]  `Doc - No Need`

### CODE REVIEW

**Review by imbajin — APPROVED** [review]
Thanks~

But too big to review the details in UI, search it in local IDE soon

**Comment by SunnyBoy-WYH** [comment]
> Thanks~
> 
> But too big to review the details in UI, search it in local IDE soon

Do we have any pre-check tool to vertify it?  so we can dont need check until release landing.

**Comment by simon824** [comment]
We need also declare the LICENSE information of the dependency in the [./hugegraph-dist/release-docs/LICENSE](https://github.com/apache/incubator-hugegraph/blob/release-1.2.0/hugegraph-server/hugegraph-dist/release-docs/LICENSE)

File: hugegraph-server/hugegraph-dist/release-docs/licenses/LICENSE-java-cup-runtime.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+
+The authors and their employers disclaim all warranties with regard to this software, including all implied warranties of merchantability and fitness. In no event shall the authors or their employers be liable for any special, indirect or consequential damages or any damages whatsoever resulting from loss of use, data or profits, whether in an action of contract, negligence or other tortious action, arising out of or in connection with the use or performance of this software.
+This is an open source license. It is also GPL-Compatible (see entry for "Standard ML of New Jersey"). The portions of CUP output which are hard-coded into the CUP source code are (naturally) covered by this same license, as is the CUP runtime code linked with the generated parser.
```
Comment (simon824) [inline hugegraph-server/hugegraph-dist/release-docs/licenses/LICENSE-java-cup-runtime.txt:8]: It seems that it is not clear from the file what license this dependency is based on. And this license contains `GPL` keywords, which should be noted and removed.

Comment (imbajin) [reply hugegraph-server/hugegraph-dist/release-docs/licenses/LICENSE-java-cup-runtime.txt:8]: > It seems that it is not clear from the file what license this dependency is based on. And this license contains `GPL` keywords, which should be noted and removed.

consider remove the unnecessary NOTICE part from upstream

**Review by simon824 — APPROVED** [review]

**Review by imbajin — APPROVED** [review]

**Pull request merged**